package.json

```diff
@@ -89,7 +89,7 @@
   "author": "Balena Inc. (https://balena.io/)",
   "license": "Apache-2.0",
   "engines": {
-    "node": ">=12 <16"
+    "node": ">=14 <16"
```

Comment:
Do we also need to update `    "@types/node": "^12.20.42",` on line 148?

Reply:
Makes sense

Reply:
I also found some other code comments that refer to `12.20.42` that can probably be cleaned up.

Reply:
Converted back to draft to fix the code comments

Reply:
Resolved here https://github.com/balena-io/balena-cli/pull/2571#discussion_r1060770099